### ISSUE TITLE
Problem: should an error occur on users' machines there's no way to know about it

### ISSUE THREAD

**Issue by sjmackenzie** [fractalide/fractalide#101]
Solution: allow users to send home logs...

**Comment by gsovereignty** [comment]
When people casually test this out they're not going to send errors, they'll just forget about the project and put it in the "too hard" basket. You should send error logs by default unless the user _explicitly_ turns them off, you're going to need all the help you can get when it comes to making it work well in the wild.

**Comment by sjmackenzie** [comment]
Yeah, opt out is probably the way to go.

**Comment by clacke** [comment]
Just make it prominent somewhere that you're phoning home, or people can get pretty upset.

**Comment by clacke** [comment]
And consider what kind of information you will be sending. The best way to not have an embarrassing leak of user info is to not hold the information in the first place.

**Comment by sjmackenzie** [comment]
Yeah I dislike these kinds of phone home apps, but I sympathize for its need. I suppose a good balancing act is strong anonymity, as you suggested :-)
